apps/bleskomat-server/data/entrypoint.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+export BLESKOMAT_SERVER_ADMIN_PASSWORD=$(npm run --silent generate:adminPasswordHash -- "$APP_PASSWORD" | xargs)
+
+exec "/usr/src/app/docker-entrypoint.sh $@"
```

Comment:
Does this file exist inside the container? It's failing for me with:

```
/data/entrypoint.sh: line 5: /usr/src/app/docker-entrypoint.sh : No such file or directory
```

Reply:
Ah it's cos you're quoting the whole lot so you're actually trying to execute "/usr/src/app/docker-entrypoint.sh " when `$@` is unset (notive the trailing space).

This should be what you want:

```suggestion
exec "/usr/src/app/docker-entrypoint.sh" $@
```